services/liquibase/main-db/changelog/changelogs/2025-04-19-1840-init-schema.yaml

```diff
@@ -0,0 +1,170 @@
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: user_id, type: uuid }
+              - column: { name: ingredient_id, type: uuid }
+
+        # administrators
+        - createTable:
+            tableName: administrators
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false } }
+
+        # review
+        - createTable:
+            tableName: review
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: recipe_id, type: uuid }
+              - column: { name: user_id, type: uuid }
+              - column: { name: rating, type: float(2) }
+              - column: { name: description, type: varchar(2000) }
+              - column: { name: has_photos, type: boolean }
+
+        # photo_urls
+        - createTable:
+            tableName: photo_urls
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false } }
+              - column: { name: photo_url, type: varchar(512) }
+
+        # user_dietary_restrictions
+        - createTable:
+            tableName: user_dietary_restrictions
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: ingredient_tags_id, type: uuid }
+              - column: { name: user_id, type: uuid }
+
+        # user_profiles
+        - createTable:
+            tableName: user_profiles
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: user_id, type: uuid }
+              - column: { name: photo_url_id, type: uuid }
+              - column: { name: description, type: varchar(2000) }
+
+        # user_cookbook_recipes
+        - createTable:
+            tableName: user_cookbook_recipes
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: recipe_id, type: uuid }
+              - column: { name: user_id, type: uuid }
+              - column: { name: is_favorite, type: boolean }
+
+        # review_photos
+        - createTable:
+            tableName: review_photos
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: review_id, type: uuid }
+              - column: { name: photo_id, type: uuid }
```

Comment:
both foreign keys, id should be removed (i think) and the primary key should be review_id and photo_id connected